auto_submit/test/requests/check_pull_request_test.dart

```diff
@@ -297,47 +377,192 @@ void main() {
     });
 
     test('Empty validations do not merge', () async {
```

Comment:
What's the difference between this one and test `Does not fail with null checks`?

Reply:
Hi Keyong, I think for `Empty validations do not merge` , both checkRuns and statuses are empty. I will keep just this one.